nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,59 @@
+/*const { response } = require("express");*/
+  const express = require("express");
+  const app = express();
+  module.exports = app;
+
+// import data here
+  const meals = require("./data/meals");
+  const reviews = require("./data/reviews")
+  const reservations = require("./data/reservations")
+
+// this is where you will be adding your routes
+app.get("/", async (request, response) => {
+  response.send("Meal Sharing Web App");
+});
+// task 1
+app.get("/meals", async(request, response) => {
+  const reviewedMeals = meals.map((meal) => {
+  meal.reviews = reviews.filter((review) => review.mealId === meal.id);
```

Comment:
Just a quick comment, try to avoid modifying input arguments like this, instead create new objects to return, like:
```js
return {
  ...meal,
  reviews: reviews.filter((review) => review.mealId === meal.id)
};
```